twoliter/embedded/Dockerfile.dockerignore

```
@@ -3,9 +3,10 @@
 /build/*
 !/build/rpms/
 /build/rpms/*
-!/build/rpms/*.rpm
-/build/rpms/*-debuginfo-*.rpm
-/build/rpms/*-debugsource-*.rpm
+!/build/rpms/*/*.rpm
+/build/rpms/*/*-debuginfo-*.rpm
+/build/rpms/*/*-debugsource-*.rpm
+!/build/kits/
```

Maybe keep the earlier exclusions for now, with a comment that we can deprecate them eventually. Otherwise the build context will get very large for anyone that has existing files in `build/rpms` until they clean their checkout.
```suggestion
!/build/rpms/*.rpm
/build/rpms/*-debuginfo-*.rpm
/build/rpms/*-debugsource-*.rpm
!/build/rpms/*/*.rpm
/build/rpms/*/*-debuginfo-*.rpm
/build/rpms/*/*-debugsource-*.rpm
!/build/kits/
```

---

I've committed this suggestion, but don't we actually want to ignore `/build/rpms/*.rpm` to solve that problem?